src/code/code_80140CE0.c

```diff
@@ -0,0 +1,86 @@
+#include "global.h"
+
+extern Gfx D_0E0001C8[];
+
+Gfx D_801C5DD0[] = {
+    gsDPSetOtherMode(G_AD_PATTERN | G_CD_MAGICSQ | G_CK_NONE | G_TC_CONV | G_TF_POINT | G_TT_NONE | G_TL_TILE |
+                         G_TD_CLAMP | G_TP_NONE | G_CYC_1CYCLE | G_PM_NPRIMITIVE,
+                     G_AC_NONE | G_ZS_PRIM | G_RM_VISCVG | G_RM_VISCVG2),
+    gsSPBranchList(0x0E0002E0),
+};
+
+Gfx D_801C5DE0[] = {
+    gsDPSetOtherMode(G_AD_PATTERN | G_CD_MAGICSQ | G_CK_NONE | G_TC_CONV | G_TF_POINT | G_TT_NONE | G_TL_TILE |
+                         G_TD_CLAMP | G_TP_NONE | G_CYC_1CYCLE | G_PM_NPRIMITIVE,
+                     G_AC_NONE | G_ZS_PRIM | IM_RD | CVG_DST_CLAMP | ZMODE_OPA | FORCE_BL |
+                         GBL_c1(G_BL_CLR_FOG, G_BL_A_FOG, G_BL_CLR_MEM, G_BL_A_MEM) |
+                         GBL_c2(G_BL_CLR_FOG, G_BL_A_FOG, G_BL_CLR_MEM, G_BL_A_MEM)),
+    gsSPBranchList(0x0E0002E0),
+};
+
+Gfx D_801C5DF0[] = {
+    gsDPSetOtherMode(G_AD_PATTERN | G_CD_MAGICSQ | G_CK_NONE | G_TC_CONV | G_TF_POINT | G_TT_NONE | G_TL_TILE |
+                         G_TD_CLAMP | G_TP_NONE | G_CYC_1CYCLE | G_PM_NPRIMITIVE,
+                     G_AC_NONE | G_ZS_PRIM | IM_RD | CVG_DST_CLAMP | ZMODE_OPA | FORCE_BL |
+                         GBL_c1(G_BL_CLR_IN, G_BL_0, G_BL_CLR_MEM, G_BL_A_MEM) |
+                         GBL_c2(G_BL_CLR_IN, G_BL_0, G_BL_CLR_MEM, G_BL_A_MEM)),
+    gsSPBranchList(0x0E0002E0),
+};
+
+Gfx D_801C5E00[] = {
+    gsDPSetCombineMode(G_CC_PRIMITIVE, G_CC_PRIMITIVE),
+    gsDPSetOtherMode(G_AD_NOTPATTERN | G_CD_DISABLE | G_CK_NONE | G_TC_CONV | G_TF_POINT | G_TT_NONE | G_TL_TILE |
+                         G_TD_CLAMP | G_TP_NONE | G_CYC_1CYCLE | G_PM_NPRIMITIVE,
+                     G_AC_NONE | G_ZS_PRIM | G_RM_CLD_SURF | G_RM_CLD_SURF2),
+    gsSPDisplayList(0x0E0002E0),
+    gsDPSetOtherMode(G_AD_PATTERN | G_CD_MAGICSQ | G_CK_NONE | G_TC_CONV | G_TF_POINT | G_TT_NONE | G_TL_TILE |
+                         G_TD_CLAMP | G_TP_NONE | G_CYC_1CYCLE | G_PM_NPRIMITIVE,
+                     G_AC_NONE | G_ZS_PRIM | IM_RD | CVG_DST_CLAMP | ZMODE_OPA | FORCE_BL |
+                         GBL_c1(G_BL_CLR_IN, G_BL_0, G_BL_CLR_MEM, G_BL_A_MEM) |
+                         GBL_c2(G_BL_CLR_IN, G_BL_0, G_BL_CLR_MEM, G_BL_A_MEM)),
+    gsSPBranchList(0x0E0002E0),
+};
+
+void func_80140CE0(struct_801F8010* arg0) {
+    arg0->type = 0;
+    arg0->setScissor = false;
+    arg0->color.r = 255;
+    arg0->color.g = 255;
+    arg0->color.b = 255;
+    arg0->color.a = 255;
+}
+
+void func_80140D04(struct_801F8010* arg0) {
+}
+
+void func_80140D10(struct_801F8010* arg0, Gfx** _gfx) {
+    Gfx* gfx = *_gfx;
+
+    gDPPipeSync(gfx++);
+    gDPSetPrimDepth(gfx++, -1, -1);
+
+    if (arg0->setScissor == true) {
+        gSPDisplayList(gfx++, D_0E0001C8);
+    }
+
+    switch (arg0->type) {
+        case 1:
+            gSPDisplayList(gfx++, D_801C5DF0);
+            break;
+        case 2:
```

Comment:
```suggestion
        case 1:
            gSPDisplayList(gfx++, D_801C5DF0);
            break;
       
        case 2:
```
Spaces after breaks is the style we're going for. Also add a default case (assuming it still matches)